.run/Aur package cache.run.xml

```diff
@@ -1,12 +0,0 @@
-<component name="ProjectRunConfigurationManager">
```

Comment:
Why did you delete these run configurations? I think they're quite useful to include